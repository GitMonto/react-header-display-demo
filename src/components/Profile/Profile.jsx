import PropTypes from "prop-types";

export default function Profile({ name, image }) {
  return (
    <section className="mx-auto flex w-48 flex-col items-center pt-11">
      <h1>Photo of {name}</h1>
      <img className="rounded-full" src={image} alt={name} />
    </section>
  );
}

Profile.propTypes = {
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
};
