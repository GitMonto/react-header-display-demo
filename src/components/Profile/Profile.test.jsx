// TODO: Write a snapshot test and visually inspect it to insure that it is rendering correctly
import renderer from "react-test-renderer";
import Profile from "./Profile";

it("renders without crashing", () => {
  const rendered = renderer.create(
    <Profile name="marko" imgUrl="someimage.jpg" />
  );
  expect(rendered).toMatchSnapshot();
});
