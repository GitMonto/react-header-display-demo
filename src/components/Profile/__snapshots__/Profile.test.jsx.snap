// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<section
  className="mx-auto flex w-48 flex-col items-center pt-11"
>
  <h1>
    Photo of 
    marko
  </h1>
  <img
    alt="marko"
    className="rounded-full"
  />
</section>
`;
